CODEOWNERS

```diff
@@ -527,6 +527,7 @@
 /samples/nrf5340/empty_app_core/          @nrfconnect/ncs-si-muffin
 /samples/nrf5340/extxip_smp_svr/          @nrfconnect/ncs-pluto
 /samples/nrf54h20/empty_app_core/         @nrfconnect/ncs-aurora
+/samples/ironside_se/                     @nrfconnect/ncs-aurora
```

Comment:
Please add an entry for the new RST in this section : https://github.com/nrfconnect/sdk-nrf/pull/24028/files#diff-fcf14c4b7b34fe7a11916195871ae66a59be87a395f28db73e345ebdc828085bR561 (ncs-aurora-doc). Also, missing changelog entry.

Reply:
done